GitCommands/Git/GetAllChangedFilesOutputParser.cs

```diff
@@ -100,7 +100,8 @@ internal List<GitItemStatus> GetAllChangedFilesFromString_v1(string getAllChange
                     continue;
                 }
 
-                if (x != '?' && x != '!' && x != ' ')
+                // Skip unmerged where both are modified: Only worktree interesting.
+                if ((x != '?' && x != '!' && x != ' ') || !(x == 'U' && y == 'U'))
```

Comment:
That was a a bigger change, to be consistent
Also modified the checks slightly, still ignoring unexpected data, required to ignore error printouts (this can maybe be changed when error output is no longer parsed, separate change).

Reply:
Thank you.